tests/test_sims.py

```diff
@@ -642,6 +661,36 @@ def test_bad_inputs(self):
             wtargs = [[1, 2], [1, 1]]
             sbm(self.n, self.Psy, wt=wt, wtargs=wtargs)
 
+        with self.assertRaises(TypeError):
+            # Check that the paramters are a dict
+            dc = np.random.uniform
+            dcargs = [1,2]
+            sbm(self.n, self.Psy, dc=dc, dcargs=dcargs)
+
+        with self.assertRaises(ValueError):
+            # There are non-numeric elements in p
+            dc = ['1',1]
+            sbm(self.n, self.Psy, dc=dc)
+
+        with self.assertRaises(ValueError):
+            # dc must have size len(n)
```

Comment:
Dc should have size sum(n) is what you wrote in the docstring, right? Just curious because regardless there is one check that isn't being hit by codecov but I think this test should be covering it 

https://codecov.io/gh/neurodata/graspy/pull/123/diff#D18-377

Reply:
Yes, that's a typo in the test file. It should say "size of sum(n)". But all of the unittests run fine and pass on my machine so I am confused as to what the issue is.